### DISCUSSION TITLE
Lucid Import and Syntax in VSCode

### DISCUSSION THREAD

**Discussion by leomoyses** [adonisjs/core#1326] [Help] [answered]
![image](https://user-images.githubusercontent.com/1256366/88452585-68cc5980-ce36-11ea-9ebd-53022bcc9df6.png)

This is happening on all imports from Lucid :(

Does anyone know why and how to fix it? 

Thanks!

Edit: Using Adonis 5

**Comment by thetutlage** [comment, marked as answer]
Have you install lucid and configured it properly using the docs?

**Reply by leomoyses** [reply]
Lucid is running without problems :(

Migrations are working flawlessly, and the health check is returning with everything ok.

Also, its an honor having you answer my issue :D

**Reply by thetutlage** [reply]
So does that mean your issue has been fixed?

**Reply by leomoyses** [reply]
It has not :(
Even though running it works, everything in VSCode is with errors and no intellisense for anything lucid related.

**Reply by donatoaguirre24** [reply]
Hey, @leomoyses. Maybe your issue it's related to [this change](https://github.com/microsoft/vscode-eslint/blob/release/2.1.x/CHANGELOG.md#217) in the VSCode eslint extension.

**Reply by leomoyses** [reply]
> Hey, @leomoyses. Maybe your issue it's related to [this change](https://github.com/microsoft/vscode-eslint/blob/release/2.1.x/CHANGELOG.md#217) in the VSCode eslint extension.

Thans buddy! That fixed it!
